tarpc/Cargo.toml

```
@@ -27,14 +27,15 @@ tarpc-plugins = { path = "../plugins", version = "0.5.0" }
 
 [dev-dependencies]
 assert_matches = "1.0"
-tarpc-json-transport = { version = "0.1", path = "../json-transport" }
+tarpc-tcp-transport = { version = "0.1", path = "../tcp-transport" }
```

I mentioned this offline, but the transports are already generic over the underlying `AsyncRead + AsyncWrite` implementation. You can see that in the From impl I've highlighed below. The name should therefore not be tied only to TCP.

```suggestion
tarpc-transport = { version = "0.1", path = "../transport" }
```